base/common/src/main/java/com/netscape/certsrv/client/PKIConnection.java

```diff
@@ -400,4 +400,10 @@ public File getOutput() {
     public void setOutput(File output) {
         this.output = output;
     }
+
+    public void close() {
```

Comment:
Is `Connection` always created in a client? Othewise we could add `AutoCloseable` here too. I've been adding it in JSS to all classes which can be closed, just in case they're used in a context with a well-bounded lifecycle.

Reply:
So far yes, but I'm thinking we might eventually merge `PKIConnection` into `PKIClient` in the future. I've updated the patch to make both classes `AutoCloseable`.